### PR TITLE
LRQA-75138 | master 

### PR DESCRIPTION
**Pull request by weller0246**
https://issues.liferay.com/browse/LRQA-75138

### CODE REVIEW

**Comment by weller0246** [comment]
ci:test:ldap:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:stop

**Comment by weller0246** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/LDAP.macro [hunk 1]
```diff
@@ -127,6 +127,22 @@ definition {
 			userGroupName = "${userGroupName}");
 	}
 
+	@description = "Verify that the user is imported on localhost after LDAP sync, after a specified interval time"
+	macro verifyUserImportAfterImportIntervalTime {
+		if (!(isSet(intervalTime))) {
+			var intervalTime = "60000";
+		}
+
+        // Wait for the amount of the import interval time
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/macros/LDAP.macro:136]: Ticket ID is missing from the text see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/check/poshi_pause_usage_check.markdown

**Comment by weller0246** [comment]
ci:test:sf

**Comment by weller0246** [comment]
ci:test:ldap:nocompile

**Comment by weller0246** [comment]
ci:test:ldap:nocompile

**Comment by weller0246** [comment]
ci:test:ldap:nocompile

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
Review done, see my comments.
Don't forget to remove CI only commit too.

**Comment by weller0246** [comment]
ci:test:ldap:nocompile

**Comment by weller0246** [comment]
ci:test:ldap:nocompile

File: portal-web/test/functional/com/liferay/portalweb/macros/LDAP.macro [hunk 2]
```diff
@@ -127,6 +127,23 @@ definition {
 			userGroupName = "${userGroupName}");
 	}
 
+	@description = "Verify that the user is imported on localhost after LDAP sync, after a specified interval time."
+	macro verifyUserImportAfterImportIntervalTime {
+		if (!(isSet(intervalTime))) {
+			var intervalTime = "60000";
+		}
+
+		// Wait for the amount of the import interval time. Needed for LRQA-75138.
```
[the remark this answers is not in the document]
Comment (weller0246) [reply portal-web/test/functional/com/liferay/portalweb/macros/LDAP.macro:136]: Added a new **while** for checking the presence of the new user

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/LDAP.macro (hunk 2 above)
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/macros/LDAP.macro:136]: Nice solution!

**Comment by gaborlovasdotcom** [comment]
@weller0246 Please SF :)

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
Don't forget to remove CI only commit also

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**